docs/cloud/connecting.md

```diff
@@ -86,7 +86,7 @@ For Prefect Cloud, you can configure the `PREFECT_API_URL` and `PREFECT_API_KEY`
 
 <div class="terminal">
 ```bash
-$ prefect config set PREFECT_API_URL="https://api.prefect.cloud/api/accounts/[ACCOUNT-ID]/workspaces/[WORKSPACE-ID]"
+prefect config set PREFECT_API_URL="https://api.prefect.cloud/api/accounts/[ACCOUNT-ID]/workspaces/[WORKSPACE-ID]"
```

Comment:
I think the `$` is useful for two reasons:

1. It separates the command a user is expected to enter from the output they'll see
2. It shows whether you can be any user (`$`) or require root (`#`)

There have been various attempts to resolve this issue: https://github.com/PrefectHQ/prefect/pull/9204

I think one way to resolve this is to make the `$` un-selectable using the `user-selection` CSS style, as someone suggested in the other PR: https://github.com/PrefectHQ/prefect/pull/9204#issuecomment-1516566973

I also think it's reasonable to use this approach for now if it's quicker. BTW the line below this comment also has a `$` that needs to be removed

Reply:
One thing to perhaps consider is that `$` and `#` make us Windows users feel like second class citizens 😂.

We're used to it, especially when working with startups and open source. So it's certainly not a deal breaker. 

OTOH, Windows users are disproportionately likely to work at companies with nice, big enterprise budgets, so little things that make them feel welcome might be useful 💰💰💰.

Fwiw, I don't think any commands in the docs need to be run as root if they're run by an account with permission to use everything a flow/worker/CLI command tries to access.

But as a developer who likes copying commands from docs, pretty much anything you do is fine as long as I don't have to manually delete leading characters after pasting a copied command😀.